news/2023/2023-12-10-osu-world-cup-2023-concludes.md

```diff
@@ -0,0 +1,406 @@
+
+With nothing but bragging rights at stake, All Stars Blue picked into [xi - FREEDOM DiVE](https://osu.ppy.sh/beatmapsets/39804)... and let's just say [Ekoro](https://osu.ppy.sh/users/284905) took advantage of the fact that EZ and FL were allowed by the referee on Free Mod.
+
+And his team looked unstoppable, having gained 6 points in a row with team red having nil to say about it. Two thirds of the way through [Rohi - Gllensista](https://osu.ppy.sh/beatmapsets/2091797#osu/4384679) however, team blue was holding a 200k point lead and looked to secure the sweep — but in the blink of an eye, the whole lobby except [scylla](https://osu.ppy.sh/users/9405745) collapsed. All Stars Red managed to keep the match alive a little longer before [Chicony](https://osu.ppy.sh/users/5199332) and [Kingling](https://osu.ppy.sh/users/7010761) carried team blue across the finish line on [FELT - Puppet in the Dark](https://osu.ppy.sh/beatmapsets/326636#osu/725198), ending in a 7–3.
+
+---
+
+Unfortunately there was no showmatch of the Grand Finals pool due to internal issues, but hopefully the actual matches made up for it.
+
+## osu! originals
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+Following the growth of osu!'s [Featured Artists](/wiki/People/Featured_Artists), it should be no surprise that this year's osu! World Cup coordinated the production of original music (which we call [osu! originals](/wiki/osu!_originals)) specifically for its mappools.
+
+Despite these expectations, absolutely no one was prepared for a massive haul of **20** new tracks, along with original artworks designed by members of the osu! community for the maps' backgrounds.
+
+To honour the outstanding creative talent behind these maps, we've neatly laid them out for you below! Click on any banner below to visit its respective map, where you'll be able to admire both the song and its background in their full glory.
+
+### Round of 32
+
+[![](/wiki/shared/news/2023-12-10-osu-world-cup-2023-concludes/Darkness-Parade.jpg)](https://osu.ppy.sh/beatmapsets/2078577#osu/4352411)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/359/Songs/cygnus_-_Darkness_Parade.mp3">
+    </audio>
+</div>
+
+### Round of 16
+
+[![](/wiki/shared/news/2023-12-10-osu-world-cup-2023-concludes/Ebb-Tide.jpg)](https://osu.ppy.sh/beatmapsets/2082040#osu/4360358)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/279/Songs/Toromaru_-_Ebb_Tide.mp3">
+    </audio>
+</div>
+
+### Quarterfinals
+
+[![](/wiki/shared/news/2023-12-10-osu-world-cup-2023-concludes/Wandered.jpg)](https://osu.ppy.sh/beatmapsets/2085296#osu/4368472)
+
```

Comment:
Is it intended that some audio previews are missing?

Reply:
"Sydosys - Wandered" and "Sorry about my face - The last straw" were skipped because they don't have featured artist pages. no idea if they're supposed to be fa at some point